src/coreclr/tools/aot/ILCompiler.Diagnostics/PdbWriter.cs

```diff
@@ -217,7 +207,9 @@ private void WritePDBDataHelper(string dllPath, IEnumerable<MethodInfo> methods)
             // Delete any preexisting PDB file upfront, otherwise CreateNGenPdbWriter silently opens it
             File.Delete(_pdbFilePath);
 
-            CreateNGenPdbWriter(dllPath, _pdbFilePath, out _ngenWriter);
+            var comWrapper = new ILCompilerComWrappers();
+            CreateNGenPdbWriter(dllPath, _pdbFilePath, out var pdbWriterInst);
+            _ngenWriter = (ISymNGenWriter2)comWrapper.GetOrCreateObjectForComInstance(pdbWriterInst, CreateObjectFlags.UniqueInstance);
```

Comment:
Does this get disposed anywhere?

Reply:
I tried to remove the explicit Marshal.Free (since I presume that's not something we want to call anymore?) but I forgot to add a `using` for the wrapper. I've added that now -- honestly I'm not completely understanding the appropriate lifetimes for these things, so this is just guessing now.